astro/src/content/docs/_shared/_setup-wizard.mdx

```diff
@@ -14,34 +14,59 @@ Once the setup wizard is complete you will be logged into FusionAuth using the c
 
 <img src="/img/docs/get-started/download-and-install/setup-wizard.png" alt="Setup Wizard" width="1200" role="shadowed" />
 
-#### Administrator account
+#### Admin user
 This will be the first account to be created in FusionAuth and this user will be assigned the `admin` role the FusionAuth application. Additional
 users can be assigned the `admin` role at a later time.
 
-#### FusionAuth license
+#### License agreement and communications
+
 To complete the setup wizard you must toggle the Accept button after reading the FusionAuth license agreement. You won't find any surprise strings attached here, just the normal lawyer speak for CYA.
 
-#### Stay informed
 Optionally stay informed by opting-in to receive FusionAuth technical announcements, tutorials and other technical focused emails. You may opt-out at any time in the future, or opt-in at a later date by visiting [fusionauth.io](/) and finding the "Subscribe for updates" form at the bottom of the page.
 
-## Complete Setup
+Your feedback on how you learned about us helps us better serve our customers.
+
+## Complete First Time Setup
 
-Once the Setup Wizard has logged you into FusionAuth you may begin using FusionAuth. On the dashboard you will see the remaining setup items.
-This includes setting up an API key, creating an Application and completing the SMTP Email configuration.
+Once the Setup Wizard has logged you into FusionAuth you may begin using FusionAuth. On the dashboard you will see a panel for new users.
+Clicking Setup will guide you through creating an Application, adding an API key, configuring the email server, and optionally activating your license.
 
-<img src="/img/docs/get-started/download-and-install/complete-setup.png" alt="Complete Setup" width="1200" role="shadowed" />
+<img src="/img/docs/get-started/download-and-install/first-time-setup-panel.png" alt="First Time Setup Panel" width="1200" role="bottom-cropped" />
```

Comment:
It's been deemed that First-time Setup is the grammatically correct way of writing this. We could either leave this for now since it matches what's in the app and screen shots, or we could update it in the headings now.